.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     - name: Setup Python
       uses: actions/setup-python@v4
       with:
-        python-version: 3.x
```

Comment:
I think if we pin this to an older release, we're less likely to miss this in future.

Reply:
Honestly, it's better to start implementing https://github.com/aio-libs/aiohttp/pull/7054#issuecomment-1358515510 rather than guessing which Python version would catch most problems. Any version from the supported runtime range may turn out to have differing typing info, and we wouldn't catch that…

Reply:
![image](https://user-images.githubusercontent.com/121081453/211444158-9559fa55-01e7-46aa-ab88-a126b9af53bf.jpeg)